src/ActionBuilder/Cart/SetLineItemExternalPrice.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace BestIt\CommercetoolsODM\ActionBuilder\Cart;
+
+use BestIt\CommercetoolsODM\Helper\PriceHelperTrait;
+use BestIt\CommercetoolsODM\Mapping\ClassMetadataInterface;
+use Commercetools\Core\Model\Cart\Cart;
+use Commercetools\Core\Model\Cart\LineItem;
+use Commercetools\Core\Request\AbstractAction;
+use Commercetools\Core\Request\Carts\Command\CartSetLineItemPriceAction;
+
+/**
+ * Builds the action to change cart item external price which don't updated by recalculate actions
+ *
+ * @author Michel Chowanski <michel.chowanski@bestit-online.de>
+ * @package BestIt\CommercetoolsODM\ActionBuilder\Cart
+ */
+class SetLineItemExternalPrice extends CartActionBuilder
+{
+    use PriceHelperTrait;
+
+    /**
+     * Constant for external price mode
+     *
+     * @var string
+     */
+    const MODE_CONSTANT = 'Commercetools\Core\Model\Cart\LineItem::PRICE_MODE_EXTERNAL_PRICE';
+
+    /**
+     * A PCRE to match the hierarchical field path without delimiter.
+     *
+     * @var string
+     */
+    protected $complexFieldFilter = 'lineItems/([^/]+)';
```

Comment:
This filter matches EVERY attribute of every line item and should be enhanced asap! (And every other occurrence of it.)